detections/endpoint/windows_explorer_exe_spawning_powershell_or_cmd.yml

```diff
@@ -0,0 +1,54 @@
+name: Windows Explorer.exe Spawning PowerShell or Cmd
+id: 593854c5-2182-49dd-9f31-18ef697445b9
+version: 1
+date: '2025-03-24'
+author: Michael Haag, AJ King, Splunk, Jesse Hunter, Splunk Community Contributor
+status: production
+type: Anomaly
```

Comment:
I don't think this is an Anomaly, can we change it to hunting? Seems like it would extremely noisy to alert on